ui/app/AppLayouts/Onboarding2/pages/CreateKeycardProfilePage.qml

```diff
@@ -0,0 +1,106 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtGraphicalEffects 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Popups 0.1
+
+import AppLayouts.Onboarding2.controls 1.0
+
+OnboardingPage {
+    id: root
+
+    title: qsTr("Create profile on empty Keycard")
+
+    signal createKeycardProfileWithNewSeedphrase()
+    signal createKeycardProfileWithExistingSeedphrase()
+
+    pageClassName: "CreateKeycardProfilePage"
+
+    contentItem: Item {
+        ColumnLayout {
+            width: parent.width
+            anchors.centerIn: parent
+
+            StatusBaseText {
+                Layout.fillWidth: true
+                text: root.title
+                font.pixelSize: 22
+                font.bold: true
+                wrapMode: Text.WordWrap
+                horizontalAlignment: Text.AlignHCenter
+            }
+            StatusBaseText {
+                Layout.fillWidth: true
+                text: qsTr("You will require your Keycard to log in to Status and sign transactions")
```

Comment:
This sentence sounds a bit weird. I'm not sure what we're trying to tell the user